src/StreamJsonRpc.Tests/JsonRpcTests.cs

```diff
@@ -682,6 +829,22 @@ private static void SendObject(Stream receivingStream, object jsonObject)
         receivingStream.Write(buffer, 0, buffer.Length);
     }
 
+    private void ReinitializeRpcWithoutListening()
```

Comment:
Regular initialization can now call into ReinitializeRpcWithoutListening() followed by StartListening()

Reply:
True. But then we wouldn't be testing the `Attach` method.